homeassistant/components/transmission/__init__.py

```diff
@@ -109,7 +109,7 @@ def update_unique_id(
     ) -> dict[str, Any] | None:
         """Update unique ID of entity entry."""
         match = re.search(
-            f"{config_entry.data[CONF_HOST]}-{config_entry.data[CONF_NAME]} (?P<name>.+)",
+            f"{config_entry.data[CONF_HOST]}-{config_entry.data.get(CONF_NAME)} (?P<name>.+)",
```

Comment:
Can we assume if a config entry doesnt have name that we don't need a migration? Can we return None if CONF_NAME is not in the config entry data?

Reply:
Done